bin/GameIndex.dbf

```diff
@@ -9607,12 +9668,15 @@ Serial = SLES-51227
 Name   = Tomb Raider - Angel of Darkness
 Region = PAL-M10
 Compat = 5
-[patches]
-	// 54AD76D7
-	comment=- This gamedisc supports multiple languages and widescreen.
-	comment=- Language & widescreen selection is done through the BIOS configuration.
-	comment=- Implementation requires Full boot - "Boot CDVD (full)".
-	comment=- Full boot is recommended anyway for this gamedisc to avoid textproblems.
+[patches = 54AD76D7]
```

Comment:
crc change should be excluded as well, there's probably a reason why it's only mentioned in a comment.

Reply:
Oh i see, thought it still worked even with commented out, change done